zerver/views/development/integrations.py

```diff
@@ -84,10 +85,9 @@ def get_fixtures(request: HttpRequest, integration_name: str = REQ()) -> HttpRes
         fixture_path = os.path.join(fixtures_dir, fixture)
         with open(fixture_path) as f:
             body = f.read()
-        try:
+        # The file extension will be used to determine the type.
+        with suppress(orjson.JSONDecodeError):
```

Comment:
Did the fixer manage to move the comments? If so, that's pretty cool!

Reply:
No, I did this fix manually. In general Ruff tries to give an error but not a fix if there isn’t an obvious place to move a comment.